docker-entrypoint.sh

```diff
@@ -94,21 +94,31 @@ if [[ "$MONGODB_DATABASE" ]]; then
    JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .mongodb.database=\"$MONGODB_DATABASE\""
 fi
 
-if [[ "$REDIS_HOST" ]]; then
+if [ -z "$REDIS_HA_NAME" ]; then
+    REDIS_HA_NAME='mymaster'
+fi
+
+if [[ "$REDIS_SENTINELS" ]]; then
+    JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .localCache.name=\"$REDIS_HA_NAME\""
+    JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .localCache.sentinels=\"$REDIS_SENTINELS\""
+elif [[ "$REDIS_HOST" ]]; then
     JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .localCache.host=\"$REDIS_HOST\""
     JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .localCache.port=6379"
 fi
 
-if [[ "$REDIS_PORT" ]]; then
+if [[ "$REDIS_PORT" ]] && [[ ! "$REDIS_SENTINELS" ]]; then
     JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .localCache.port=$REDIS_PORT"
 fi
 
-if [[ "$REDIS_HA_HOST" ]]; then
+if [[ "$REDIS_SENTINELS" ]]; then
```

Comment:
should we combine this with L#101?

Reply:
I left them separate so they can be configured separately. redis uses REDIS_HA_* and localCache uses REDIS_*